backend/api/Areas/Leases/Models/Search/LeaseFilterModel.cs

```diff
@@ -136,7 +136,7 @@ public override bool IsValid()
                 || !string.IsNullOrWhiteSpace(PinOrPid)
                 || !string.IsNullOrWhiteSpace(LFileNo)
                 || !string.IsNullOrWhiteSpace(Address)
-                || !string.IsNullOrWhiteSpace(LeaseStatusType)
+                || (LeaseStatusTypes.Count != 0)
```

Comment:
nit: > is probably more correct, although negatives shouldn't be an issue here realistically.